src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,56 @@
+package seedu.addressbook.data;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+import java.util.HashMap;
+
+public class Tagging {
```

Comment:
This class is not called by any other classes. According to the diagram, this class should be instantiated and kept track of by AddressBook.